### PR TITLE
[WIP]extending button's locators for upcomming 5.8

### PR DESCRIPTION
**Pull request by lkhomenk**
our framework was failing on adding providers - buttons were updated there

### CODE REVIEW

**Comment by lkhomenk** [comment]
this is for 5.8 but it doesn't break our current automation

File: cfme/web_ui/form_buttons.py [hunk 1]
```diff
@@ -28,13 +28,15 @@ class FormButton(Pretty):
...
         TYPE_CONDITION = (
-            "(contains(@class, 'button') or contains(@class, 'btn') or contains(@src, 'button'))"
+            "(contains(@class, 'button') or contains(@class, 'btn') or contains(@src, 'button') "
+            "or contains(@class,'ng-isolate-scope'))"
         )
         DIMMED = "(contains(@class, 'dimmed') " \
             "or contains(@class, 'disabled') " \
-            "or contains(@class, 'btn-disabled'))"
+            "or contains(@class, 'btn-disabled')" \
+            "or contains(@class, 'ng-isolate-scope ng-hide'))"
```
Comment (jkrocil) [inline cfme/web_ui/form_buttons.py]: This is not correct - at least not for Infra Provider Add page. `.is_dimmed` check doesn't work because of that.

**Comment by mkoura** [comment]
The old button is still there, so why trying to use the miq-button? The original button is now missing the alt text, but text() can be used instead.
I tried to fix it without much testing, just to be able to run my tests - https://github.com/ManageIQ/integration_tests/pull/4360

**Comment by jkrocil** [comment]
Superseded by #4360, closing.

**Pull request closed without merging**